yarn2-plugins/prune-dev-dependencies/tests/fixtures/using-pnp-linker/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "using-pnp-linker",
+  "packageManager": "yarn@3.2.0",
```

Comment:
Totally unrelated to this PR, but I guess we should start thinking about supporting this key in our buildpack.